cs104/interface.go

```diff
@@ -19,6 +19,7 @@ type ServerHandlerInterface interface {
 	ResetProcessHandler(asdu.Connect, *asdu.ASDU, asdu.QualifierOfResetProcessCmd) error
 	DelayAcquisitionHandler(asdu.Connect, *asdu.ASDU, uint16) error
 	ASDUHandler(asdu.Connect, *asdu.ASDU) error
+	ConnectionEventHandler(asdu.Connect, bool)
```

Comment:
OnConnectionHandler and OnDisconnectionHandler is better

Reply:
Yes It is better and much more understandable.

Reply:
@ecoshub can you resolve it? then I will merge the request.